Dikke Ploaten/Controller/BaseAlbumListTableViewController.swift

```diff
@@ -35,7 +35,13 @@ class BaseAlbumListTableViewController : UITableViewController {
 	// MARK: - TableView
 	
 	override func numberOfSections(in tableView: UITableView) -> Int {
-		return albumSection.count
+		if(albumSection.count == 0){
+			self.tableView.setEmptyMessage("Je hebt nog geen albums toegevoegd.\nGa naar de zoekfunctie en voeg albums toe!")
```

Comment:
# 😱

Geen functionaliteit gaan doen in een getter van informatie. Het enige dat deze methode mag doen is het ophalen van hoeveel items er in de sectie zitten.

Vraagje: wanneer ben je zeker hoeveel albums er zijn? Stel het no data label daar in.